server/server.js

```diff
@@ -1,10 +1,24 @@
 const express = require('express');
-const port = 2001;
-
+const path = require('path');
+const bodyParser = require('body-parser');
+const cors = require('cors');
+const db = require('../db/schema.js');
 
+const port = 2001;
+const app = express();
 
+app.use(cors());
+app.use(bodyParser.json());
+app.use(express.static(path.join(__dirname, '../public')));
 
+app.get('/photos', (req, res) => {
+  db.Photo.find({ restuarantId: req.body.id }, (err, urls) => {
```

Comment:
I think you should separate the concerns a little more here. The query is being made directly inside the router. Instead of importing the model you could write a function that makes the query, then import that function.

Reply:
@rafeautie Thats a good idea! I've written a separate query function and imported it instead